signals.mss

```diff
@@ -1341,7 +1341,8 @@ Format details:
   /* NL departure light signals */
   /******************************/
   ["feature"="NL"]["departure_form"="light"],
-  ["feature"="NL:VL"]["departure_form"="light"] {
+  ["feature"="NL:VL"]["departure_form"="light"],
+  ["feature"="NL:VL"]["departure_form"="NL:light"] {
```

Comment:
I don't think this is correct tagging after all. Looks more like a tagging mistake that should be fixed in OSM.

Reply:
Done in mass edit https://www.openstreetmap.org/changeset/142925743#map=10/52.2030/5.8392